src/core/store/store.ts

```diff
@@ -2,12 +2,22 @@ import { createStore } from '@reatom/core';
 import { createDevtoolsLogger } from '~utils/debug/reatom-redux-devtools';
 import { appMetrics } from '~core/metrics';
 
+// enable with localStorage.setItem('KONTUR_DEBUG', 'true')
+const KONTUR_DEBUG = !!globalThis.window?.localStorage.getItem('KONTUR_DEBUG');
+// enable with localStorage.setItem('KONTUR_TRACE_ERROR', '_error')
+const KONTUR_TRACE_TYPE = globalThis.window?.localStorage.getItem('KONTUR_TRACE_TYPE');
+
 function configureStore() {
   const devtoolsLogger = createDevtoolsLogger();
```

Comment:
seems a bit weird that devtoolsLogger is created and never used without VITE_REDUX_DEV_TOOLS flag. 
maybe put in right before usage? `return createStore({`